indico/modules/events/timetable/util.py

```diff
@@ -196,3 +216,53 @@ def render_session_timetable(session, timetable_layout=None, management=False):
     event_info = serialize_event_info(session.event_new)
     tpl = get_template_module('events/timetable/_timetable.html')
     return tpl.render_timetable(timetable_data, event_info, timetable_layout=timetable_layout, management=management)
+
+
+def get_top_timetable_session_blocks(event, day):
+    """Returns a list of event top-level session blocks for the
+       given `day`
+    """
+    return (event.timetable_entries
+            .filter(db.cast(TimetableEntry.start_dt.astimezone(event.tzinfo), db.Date) == day.date())
+            .filter(TimetableEntry.type == TimetableEntryType.SESSION_BLOCK, TimetableEntry.parent_id.is_(None))
+            .all())
+
+
+def move_timetable_entry(entry, parent_entry=None, move_to_day=None):
+    """Moves the `entry` to another session or top-level
+       timetable
```

Comment:
i'd move it back to the prev line. the 71 chars limit is not THAT strict (and when you do need to wrap, the opening `"""` go into their own line)